src/lavatory/commands/purge.py

```diff
@@ -15,46 +16,69 @@
 @click.option(
     '--dryrun/--nodryrun', default=True, is_flag=True, help='Dryrun does not delete any artifacts. On by default')
 @click.option('--default/--no-default', default=True, is_flag=True, help='If false, does not apply default policy')
-@click.option('--repo', default=None, multiple=True, required=False, 
-              help='Name of specific repository to run against. Can use --repo multiple times. If not provided, uses all repos.')
+@click.option(
+    '--repo',
+    default=None,
+    multiple=True,
+    required=False,
+    help='Name of specific repository to run against. Can use --repo multiple times. If not provided, uses all repos.')
 def purge(ctx, dryrun, policies_path, default, repo):
     """Deletes artifacts based on retention policies"""
     artifactory = Artifactory(repo_name=None)
-    plugin_source = setup_pluginbase(extra_policies_path=policies_path)
-
-    before_repo_data = artifactory.list()
+    before_purge_data = artifactory.list()
     if repo:
         all_repos = repo
     else:
-        all_repos = before_repo_data.keys()
-    
+        all_repos = before_purge_data.keys()
```

Comment:
I'd change `all_repos` to `selected_repos`. The name makes me believe I'm getting all the repos, where, in reality I may not.